external/java8-tests/src/test/java/test/org/apache/spark/java8/Java8RDDAPISuite.java

```diff
@@ -303,43 +301,6 @@ public void zipPartitions() {
   }
 
   @Test
-  public void accumulators() {
```

Comment:
I just removed this as it's a copy-paste of the Java (7) test. I figured there's not much need to keep retesting the deprecated API, but don't feel strongly about it. Easy to add this back and update it.